core/src/main/java/org/elasticsearch/index/query/GeohashCellQuery.java

```diff
@@ -93,9 +97,9 @@ public static Query create(QueryShardContext context, GeoPointFieldMapper.GeoPoi
         // because a transformation from a geohash to a point an back to the
         // geohash will extend the accuracy of the hash to max precision
         // i.e. by filing up with z's.
-        private String field;
+        private String fieldName;
         private String geohash;
-        private int levels = -1;
+        private Integer levels = null;
         private boolean neighbors;
```

Comment:
I'll add a default of `false` for neighbors but `null` is actually the sensible default for `levels` as it means 'do not truncate the given geohash'. The `levels` parameter is only there to truncate the precision of the given geohash if required